firebase-firestore/src/main/java/com/google/firebase/firestore/remote/Datastore.java

```diff
@@ -246,8 +247,13 @@ public Task<Map<String, Value>> runAggregateQuery(
                 throw task.getException();
               }
 
+              Map<String, Value> result = new HashMap<>();
               RunAggregationQueryResponse response = task.getResult();
-              return response.getResult().getAggregateFieldsMap();
+              for (Map.Entry<String, Value> entry :
+                  response.getResult().getAggregateFieldsMap().entrySet()) {
+                result.put(aliasMap.get(entry.getKey()), entry.getValue());
```

Comment:
Consider an assert here that validates the key is in the alias map, something that would produce a usable error message if the server gave us an unexpected key in the result.